checks.d/mysql.py

```diff
@@ -1155,21 +1200,26 @@ def _query_size_per_schema(self, db):
                  GROUP BY table_schema;
         """
 
-        with closing(db.cursor()) as cursor:
-            cursor.execute(sql_query_schema_size)
+        try:
+            with closing(db.cursor()) as cursor:
+                cursor.execute(sql_query_schema_size)
+
+                if cursor.rowcount < 1:
+                    raise Exception("Failed to fetch records from the information schema 'tables' table.")
 
-            if cursor.rowcount < 1:
-                raise Exception("Failed to fetch records from the information schema 'tables' table.")
+                schema_size = {}
+                for row in cursor.fetchall():
+                    schema_name = str(row[0])
+                    size = long(row[1])
 
-            schema_size = {}
-            for row in cursor.fetchall():
-                schema_name = str(row[0])
-                size = long(row[1])
+                    # set the tag as the dictionary key
+                    schema_size["schema:{0}".format(schema_name)] = size
 
-                # set the tag as the dictionary key
-                schema_size["schema:{0}".format(schema_name)] = size
+                return schema_size
+        except Exception as e:
+            self.log.warning("Avg exec time performance metrics unavailable at this time: %s", e)
```

Comment:
in the message, you probably meant `Size per schema metrics unavailable...` instead of `Avg exec time ...`

nitpick: same thing in the description of the `_query_size_per_schema` method